src/dataexplorer/dataexplorer-application/src/components/CompareQuestionnaireResponses/CompareQuestionnaireResponses.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { useContext } from 'react';
+import { Row } from 'antd';
+import FecthSingleQuestionnaireResponses from './FetchSingleQuestionnaireResponses';
+import { PatientContext } from 'components/Patient/PatientContext';
+
+const CompareQuestionnaireresponses = () => {
```

Comment:
Same name as filename
```suggestion
const CompareQuestionnaireResponses = () => {
```